### PR TITLE
Signin dev complete

### PR DESCRIPTION
**Pull request by simrin051**
Files To Check

1. signin.html
2. signin.css

Desktop

image

Mobile 

image

### CODE REVIEW

File: signin.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    display: none;
+}
+
+.alert {
+    border-color: #fc8eac;
+    text-align: center;
+}
+
+.badcredentials {
+    background-color: #ffe6ea;
```
Comment (sri-rishi) [inline signin.css]: Suggestion: use variable  color naming for colors

File: signin.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                                Please enter username
+                            </div> -->
+                        </div>
+                        <div class="form-group">
+                            <input type="password" class="form-control rounded-0" placeholder="Password*" formControlName="password">
+                        </div>
+                        <div class="alert">
+                            <!--<div>
+                                Please enter password
+                            </div> -->
```
Comment (sri-rishi) [inline signin.html:35]: Suggestion: remove comment before committing and pushing

File: signin.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                        </div>
+                        <div class="authenticate col-md-12">
+                            <button type="button" class="Login btn  btn-block btn-lg col-md-12 rounded-0" (click)="login()">Login</button>
+                        </div>
+                    </form>
+                    <div class="form-group">
+                        <a href="" class="backLink">Back</a>
```
Comment (sri-rishi) [inline signin.html]: Suggestion: provide `href` attribute some value if you don't want to give value provide it `#` value

File: signin.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+
+<body>
+    <div id="MyModal" tabindex="-1" role="dialog">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" style=" color: white;"><i class="fa fa-sign-in" aria-hidden="true"></i>SIGN IN</h5>
```
Comment (sri-rishi) [inline signin.html]: Suggestion: avoid using inline styling

**Review by sri-rishi — COMMENTED** [review]
Great work, code looks cleaner, feel free to merge

File: signin.html (hunk 2 above)
Comment (simrin051) [reply signin.html:35]: These comments are for future once integrated with backend. It is commented purposefully.

**Comment by simrin051** [comment]
@sri-rishi 

Thank you for the review. I have resolved the comments.  Kindly seek your approval.

**Pull request merged**